ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json

```diff
@@ -43,8 +43,8 @@
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "ihtReferenceNumber", "Label": "IHT Reference Number", "FieldType": "Text", "SecurityClassification": "Public"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "ihtGrossValue", "Label": "Enter the gross value of the estate", "FieldType": "MoneyGBP", "SecurityClassification": "Public"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "ihtNetValue", "Label": "Enter the net value of the estate", "FieldType": "MoneyGBP", "SecurityClassification": "Public"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "extraCopiesOfGrant", "Label": "How many extra UK copies of the grant do you need?", "FieldType": "Number", "SecurityClassification": "Public"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "outsideUKGrantCopies", "Label": "How many sealed and certified copies of the grant do you need for use outside the UK?", "HintText": "Certain non-UK territories may need your client to provide a sealed and certified grant. Check if this is needed for your case, as a UK grant is still valid in some non-UK countries and jurisdictions", "FieldType": "Number", "SecurityClassification": "Public"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "extraCopiesOfGrant", "Label": "How many extra UK copies of the grant do you need?", "FieldType": "Number", "Min": "0", "SecurityClassification": "Public"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "outsideUKGrantCopies", "Label": "How many sealed and certified copies of the grant do you need for use outside the UK?", "HintText": "Certain non-UK territories may need your client to provide a sealed and certified grant. Check if this is needed for your case, as a UK grant is still valid in some non-UK countries and jurisdictions", "FieldType": "Number", "Min": "0", "SecurityClassification": "Public"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "applicationFee", "Label": "Application fee", "FieldType": "MoneyGBP", "SecurityClassification": "Public"},
```

Comment:
Very minor - would be a bit better if the Min0 property was in the same place within the json properties for both entries

Reply:
Yes - good point - definitely do that

Reply:
HOLD on - it IS ALREADY in the same place, just that the second line has a massive hint text on it - its correct as it is now